hawkular-alerts-rest-tests/src/test/resources/standalone-ha.xsl

```diff
@@ -67,6 +67,9 @@
       <replicated-cache name="publish" mode="ASYNC">
         <transaction mode="BATCH"/>
       </replicated-cache>
+      <replicated-cache name="dataIds" mode="ASYNC">
+        <transaction mode="BATCH"/>
+      </replicated-cache>
       <replicated-cache name="schema" mode="SYNC">
```

Comment:
I think one thing we can do is to protect this load in a serialized way.
As it is one-step we can lock the entire cache for this operation.
So, no matter when a node joins, no inconsistencies as this cassandra -> cache will be done in atomic bases.

+1
I will update the comments with this.